packages/server/src/server.ts

```diff
@@ -322,15 +328,73 @@ app.get(
     async (req: express.Request, res: express.Response, next: express.NextFunction) => {
         try {
             // We use the fastly geo header for determining the correct currency symbol
-            const countryCode = req.header('X-GU-GeoIP-Country-Code');
+            const countryCode = req.header('X-GU-GeoIP-Country-Code') || 'GB';
+            const countryGroupId = countryCodeToCountryGroupId(countryCode);
+            const choiceCardAmounts = await cachedChoiceCardAmounts();
             const ampVariantAssignments = getAmpVariantAssignments(req);
-            const response = await ampEpic(ampVariantAssignments, countryCode);
+            const epic = await ampEpic(ampVariantAssignments, countryCode);
+            const acquisitionData = {
+                source: 'GOOGLE_AMP',
+                componentType: 'ACQUISITIONS_EPIC',
+                componentId: epic.cta.componentId,
+                campaignCode: epic.cta.campaignCode,
+                abTest: {
+                    name: epic.testName,
+                    variant: epic.variantName,
+                },
+                referrerUrl: req.query.webUrl,
+            };
+            const ampState = {
+                ctaUrl: `${epic.cta.url}?INTCMP=${
+                    epic.cta.campaignCode
+                }&acquisitionData=${JSON.stringify(acquisitionData)}`,
+                reminder: {
+                    ...buildReminderFields(),
+                    hideButtons: false,
+                    hideReminderWrapper: true,
+                    hideSuccessMessage: true,
```

Comment:
Having the tracking, amounts, cta etc in this initial payload is really great! 👍 
Some of these reminder fields are very specific to the client-side rendering, which ideally the server wouldn't know about. Is it possible to initialise these client-side instead, or does it all have to arrive in the response?

Reply:
Yeah agree on that one - if it's possible & not too tricky it would be great for the server to not know about the client side state